ibft/ibft.go

```diff
@@ -40,29 +42,20 @@ type IBFT interface {
 // ibftImpl implements IBFT interface
 type ibftImpl struct {
 	instances      map[string]*Instance // key is the instance identifier
-	storage        storage.Storage
-	me             *proto.Node
+	ibftStorage    collections.Ibft
```

Comment:
@nivBlox seems redundant to call it ibftStorage... storage is fine no?

Reply:
we have differ types of storage (collections)  pls take a look at the pr's description (if that what you mean)

Reply:
yeah but it's one storage instance no?

Reply:
yes, one instance, with differ collections